ParisBNF/abb/BNFabb122.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Book of Sirach, Canon of the Doctors of the Church for those who Abjure, and other texts</title>
+           <title>Book of Sirach, Canon of the Doctors of the Church for those who Abjure, <foreign xml:lang="gez">Maṣḥafa falāsfā ṭabibān</foreign>, and other texts</title>
```

Comment:
I think 'abjure' should not be capitalised. Semantically, it is perhaps more convenient to use the more frequent 'renounce'.